docs/conf.py

```diff
@@ -24,7 +24,7 @@
 year = u'2015'
```

Comment:
Is this still accurate? If you create new files/functions the copyright year would be the present year, 

Also, what does this file do? Does it put your copyright on all the files? What if someone wanted to add to your project?

Reply:
I think it just adds some stuff to the docs.  I updated the year and author list.